### PR TITLE
Feature/scrum 3/login

### PR DESCRIPTION
**Pull request by Morganh201**
References
Description:
Change Type (feat, fix, etc.): 

feat: Finished login page, fully tested and follows MVP design pattern

Code Review Checklist:

- [] Tested feature manually via emulator
- [] Created and passed all test cases
- [] Consistent formatting
- [] Clean code with no errors or warnings
- [] Adheres to SOLID principles

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -39,4 +39,6 @@
     </string>
     <string name="content_description_report">Navigates to a page to generate a report about catalog items</string>
     <string name="content_description_delete">Deletes the selected catalog items</string>
+    <!-- TODO: Remove or change this placeholder text -->
```
Comment (JaydenFloof) [inline app/src/main/res/values/strings.xml]: Resolve his todo comment

**Comment by David-Huynh** [comment]
@JaydenFloof can you test select on this branch with your database with mine the select button doesn't change

**Comment by JaydenFloof** [comment]
> @JaydenFloof can you test select on this branch with your database with mine the select button doesn't change

Select works fine for me even when I don't switch the database link to mine.

**Review by JaydenFloof — APPROVED** [review]
Code has been reviewed and tested

**Pull request merged**